src/Widgets/SliderWidget/SliderWidgetEditingConfig.ts

```diff
@@ -11,6 +11,10 @@ provideEditingConfig(SliderWidget, {
       title: 'Autoplay?',
       description: 'Default: No',
     },
+    autoplayInterval: {
+      title: 'Autoplay interval (in s)',
```

Comment:
* [ ] FYI we could write out "seconds" here for more clarity
* [ ] FYI Maybe there's a more editor-friendly wording instead of the technical term "autoplay interval"? "Time between slides"?